lib/services/usb-livesync-service.ts

```diff
@@ -67,8 +67,11 @@ export class UsbLiveSyncService extends usbLivesyncServiceBaseLib.UsbLiveSyncSer
 				}).future<boolean>()();
 			};
 
-			let notRunningiOSSimulatorAction = (): IFuture<void> => {
-				return this.$platformService.deployOnEmulator(this.$devicePlatformsConstants.iOS.toLowerCase());
+			let notRunningiOSSimulatorAction = (): IFuture<boolean> => {
+				return (() => {
+					 this.$platformService.deployOnEmulator(this.$devicePlatformsConstants.iOS.toLowerCase()).wait();
+					 return false;
```

Comment:
https://github.com/telerik/mobile-cli-lib/pull/506/files#diff-c8066f499a9cad66c8bae208dd04b18cR145. The problem is that `livesync` command doesn't need to sync the application if the simulator is not running, but the `test` command needs to perform livesync after the simulator get started.